### ISSUE TITLE
Usersnap Feedback - My guess is that this is an editing problem, and not the code. When I sele[...]

### ISSUE THREAD

**Issue by ascott** [participedia/usersnaps#1102]
**Sender**: pscully@clearviewconsultingllc.com
**Comment**: My guess is that this is an editing problem, and not the code. When I select the url for the German translation it says "this page cannot be found." It may be that Jaskiran will need to manually delete this message from entries.
[Open #794 in Usersnap Dashboard](https://usersnap.com/a/#/participedia-dev/p/participedia-net-9cb6d041/794)

<a href='https://usersnappublic.s3.amazonaws.com/2020-07-24/14-51/519d7216-2de9-433b-be71-ff373d56e210.png'>![Usersnap Feedback - My guess is that this is an editing problem, and not the code. When I sele(...)](https://usersnappublic.s3.amazonaws.com/2020-07-24/14-51/3ca48dc3-9648-4787-a0f6-db0de400f0e5.jpg)</a>
<a href='https://usersnappublic.s3.amazonaws.com/2020-07-24/14-51/519d7216-2de9-433b-be71-ff373d56e210.png'>Download original image</a>

**Browser**: Firefox 78 (Windows 10)
**Referer**: [https://participedia.net/case/51](https://participedia.net/case/51)
**Screen size**: 1920 x 1200 **Browser size**: 1920 x 1045

Powered by [usersnap.com](https://usersnap.com/?utm_source=github_entry&utm_medium=web&utm_campaign=product)

**Comment by jesicarson** [comment]
That's correct, @plscully (you can tell for sure by viewing the case in German, and see that there were editing prompts added there as well that are different to the English version). I think Jaskiran can use the spreadsheet that she generated for the dev team to track language versions, sort by the entries that were entered in more than one language, scan for these editing prompts, and then edit each entry to delete them.

**Comment by plscully** [comment]
@jesicarson OK - Good plan. I will coordinate with Jaskiran
